BackofficeBundle/Resources/views/BackOffice/Underscore/orchestraModalView._tpl.twig

```diff
@@ -1,15 +1,15 @@
 <div class="modal-dialog">
     <div class="modal-content">
-        <div class="modal-header">
+        <div class="modal-header clearfix">
             <button type="button" class="close" data-dismiss="modal">
                 <span aria-hidden="true">&times;</span>
                 <span class="sr-only">{{ ('open_orchestra_backoffice.modal.close') | trans() }}</span>
             </button>
+            <% if (typeof actionButtons != 'undefined') { %>
+            <%= actionButtons %>
+            <% } %>
```

Comment:
The close button is on the right side.
![screenshot from 2015-11-16 10 18 10](https://cloud.githubusercontent.com/assets/11750180/11178198/7b7e821a-8c4b-11e5-8432-44b2544c7272.png)